### DISCUSSION TITLE
Sponsor button

### DISCUSSION THREAD

**Discussion by Mamidze** [community/community#106901] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, I have been waiting for GitHub stuff member to approve my request and I have meet all the requirements but they are taking time to approve

My issue is that I currently applied to several funding programs and some are declining because I don't have SPONSOR THIS PROJECT button in my account.please help?

**Comment by gil-son** [comment, marked as answer]
Hello, how are you? I feel for that situation

Acces the [Github Support](https://support.github.com/)

Click on My Ticket :

![github04](https://github.com/community/community/assets/72712095/096b3477-cee6-4a19-9f1b-6fb2219b51a1)

and open a New Ticket:

![github05](https://github.com/community/community/assets/72712095/7413fc55-875a-498c-93eb-5c54d5ffc8d6)


God bless you!

**Comment by Mamidze** [comment]
Sorted
